src/gui/mzroll/background_peaks_update.cpp

```diff
@@ -461,30 +460,28 @@ void BackgroundPeakUpdate::updateGroups(QList<PeakGroup> &groups,
     for (PeakGroup& group : groups) {
         mavenparameters->allgroups.push_back(group);
     }
+    for (PeakGroup& group : groups) {
```

Comment:
I think we talked, that this could be one single loop?

Reply:
yeaa... but while doing I noticed.. that pull isotopes works with maven parameters.. I did not want to disturb the main window maven parameters group.. so I made a copy in the earlier loop... where in adding the functionality to same group.. maven parameters all group would not have consistent value